docs/source/cdk/cdk-deployment-personal-aws.rst

```diff
@@ -42,4 +47,4 @@ For example::
             29143fef993fc62aeb3447a224679eafa9e60eaba00aa3bfaa60f0de9f9815bb: digest: sha256:e3c8f122ade7a0c1f598b3c7bbc08488c694aa9b7279e1367227ed0d0fba6c33 size: 2628
 
         there's no prompt and it hangs there forever. However, if I hit ``Enter``, a ``y/n`` prompt appears asking if I want to deploy. So if it gets stuck at that point, hit ``Enter``, then you can select ``y``.
-10. Before committing any changes, make sure to revert your ``cdk.json`` file to its original state.
\ No newline at end of file
+10. Before committing any changes, make sure to revert your ``cdk.json`` file to its original state.
```

Comment:
🤔 what even changed here?

Reply:
You might have added a newline to the end of the file?